src/main/java/org/fundacionjala/fc/config/PropertiesReader.java

```diff
@@ -0,0 +1,124 @@
+package org.fundacionjala.fc.config;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * PropertiesReader class.
+ */
+public final class PropertiesReader {
+
+    /**
+     * FileReader attribute.
+     */
+    private static FileReader reader = null;
+
+    /**
+     * Properties attribute.
+     */
+    private static Properties property = null;
```

Comment:
Please rename this variable to `properties`, and it's not necessary defined as static.